package.json

```diff
@@ -5,7 +5,10 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "tsc && node dist/app.js"
+    "lint": "eslint . --ext .ts",
+    "start:build": "tsc -w",
+    "start:run": "nodemon dist/app.js",
+    "start": "concurrently npm:start:*"
```

Comment:
Change your start script to run TypeScript files so we can use it only for development and add your build files that goes into dist directory to gitignore